docs/developer-center/new-guides/quickstart.md

```diff
@@ -0,0 +1,75 @@
+## Quickstart
+
+Hi! Glad to see you around! This guide walks you through the common steps a data scientist has to follow in their spatial analyses and how CARTOframes can help in each of them. We will follow a real use case to show it. We encourage you to have your environment ready so you can learn how to use the library by using it. If you haven't done it yet, check this guide.
+
+### Real use case
+
+Imagine that you are a data scientist at a bikesharing company and you want to know how well the company has placed their stations. Apart from that, you would also like to recommend new spots so the company can continue growing. Before starting, you list the steps you have to follow to achieve it:
+
+- Get and explore company's data
+- Discover and get meaninful data
+- Analyse which are the best and worst spots
+- Share the results with your team
+
+Let's start!
+
+
+### Get and explore company's data
+
+In order to start your work, you ask your team where you can find data about the bike stations. A team mate sends you the following file: TODO. This file contains information about the bike stations at Arlington and you check it using a pandas dataframe.
+
+```py
+import pandas as pd
+
+arlington_file = 'arlington_bikeshare_july_agg.csv'
+df = pd.read_csv(arlington_file)
+df.head(3)
+```
+
+![Bikeshare data](../img/new-guides/quickstart/initial_dataframe.png)
+
+As you can see, reading a dataframe doesn't let you understand quickly where the stations are. You think it will be awesome to show the data in a map but before doing that, you need to transform the `intersection` column into geometries. This process is called geocoding and the CARTO platform provides an easy way to do it (you can know more about it in their own guide).
```

Comment:
By only reading a dataframe you aren't able to see at a glance where the stations are. So let's visualize it in a map! 

The first thing you have to do is to transform the intersection column into geometries. This process is called geocoding and CARTO provides an easy way to do it (you can learn more about it in this [guide about geocoding](#))